libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -111,6 +111,13 @@ const AP_Param::GroupInfo AP_MotorsUGV::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("VEC_ANGLEMAX", 13, AP_MotorsUGV, _vector_angle_max, 0.0f),
 
+    // @Param: THST_ASYM
+    // @DisplayName: Thrust Asymmetry
```

Comment:
Let's start the display name with "Motor ".

Maybe we can make the description something like this?
"Motor thrust asymmetry between forward and reverse directions.  E.g. set to 2.0 if motors provide twice as much forward thrust as reverse thrust.  Only used for skid-steering vehicles."